### ISSUE TITLE
Should not compile msgstr ""

### ISSUE THREAD

**Issue by dopesong** [cloud-walker/gulp-gettext#1]
It converts into empty strings - should leave original translation.

**Comment by cloud-walker** [comment]
gulp-gettext use https://github.com/andris9/gettext-parser under the hood.

btw, are you sure that the correct behaviour is to leave the original translation? (msgid?)

**Comment by dopesong** [comment]
Yeah it should leave original translation if string is empty. Try it with msgfmt :)
